drivers/flash/flash_mcux_flexspi_mx25um51345g.c

```diff
@@ -0,0 +1,550 @@
+/*
+ * Copyright 2020 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT	nxp_imx_flexspi_nor
```

Comment:
this will become `nxp_imx_flexspi_mx25um51345g` after updating the dts compatible

Reply:
Done.